src/main/java/org/lwjgl/openal/AL.java

```diff
@@ -70,11 +70,13 @@ public static boolean isCreated() {
     }
 
     public static void destroy() {
-        org.lwjgl3.openal.ALC10.alcDestroyContext(alcContext.context);
-        org.lwjgl3.openal.ALC10.alcCloseDevice(alcDevice.device);
-        alcContext = null;
-        alcDevice = null;
-        created = false;
+        if (created){
+            org.lwjgl3.openal.ALC10.alcDestroyContext(alcContext.context);
+            org.lwjgl3.openal.ALC10.alcCloseDevice(alcDevice.device);
```

Comment:
`alcDestroyContext` should check if `alcContext != null` first
`alcCloseDevice` should check if `alcDevice != null` first

instead of using the `created` boolean.

Reply:
Done, but I'm curious if they can be non-null when create is false. I checked the logs and it works fine.